### ISSUE TITLE
Not an issue with code, but issue none the less

### ISSUE THREAD

**Issue by glyphx** [scavicchio/Easy-Walton-Miner#6]
Reading a forum, ended up here.  Was curious how you're able to run cpu and gpu miner without vm.  

C:\Users\glyph\Desktop\Walton-GPU-64>.\walton --identity "development" --rpc --rpcaddr 127.0.0.1 --rpccorsdomain "*" --datadir "node1" --port "30304" --rpcapi "admin,personal,db,eth,net,web3,miner" --extradata "b" --ipcdisable --networkid 999 --rpcport 8546 console
INFO [04-23|13:15:21] Initialised chain configuration          config="{ChainID: 1 Homestead: 1150000 DAO: 1920000 DAOSupport: true EIP150: 2463000 EIP155: 2675000 EIP158: 2675000 Metropolis: 9223372036854775807 Engine: ethash}"
INFOFatal error: listen tcp 127.0.0.1:10240: bind: Only one usage of each socket address (protocol/network address/port) is normally permitted. [panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xc0000005 code=0x0 addr=0x0 pc=0xaa0867]

goroutine 92 [running]:
github.com/ethereum/go-ethereum/miner.startServer(0xc04213a1c0)
        D:/GoProject/src/github.com/ethereum/go-ethereum/miner/agent.go:138 +0x87
created by github.com/ethereum/go-ethereum/miner.NewCpuAgent
        D:/GoProject/src/github.com/ethereum/go-ethereum/miner/agent.go:57 +0x133

This is what I get when I try with your approach.

**Comment by scavicchio** [comment]
Hi glyphx,

This software is not meant to mine CPU and GPU at the same time. It is just meant to give an easy way to start the program and provide some basic functions like hashrate logging. 

That said, if you want to run a GPU and CPU miner at the same time, you will have to do some hex editing on the walton.exe to change the ports that the miner communicates through. Once you have done that, then you can use the files we provided - with some edits - to whatever end you like. For info on CPU + GPU mining without VM, i suggest you look at this post on reddit. 
https://www.reddit.com/r/WaltonchainMining/comments/8dxgmo/tut_how_to_create_cpugpu_miner/

**Comment by glyphx** [comment]
Thanks so much for your quick response, I know that's not the purpose of this software, I followed a forum thread here, and I assumed you were the author.  I've also created a walton automation script.  I will look at the source of yours as well as you did some fancier stuff and duplicated stdout.   Mine might run well on crappy systems and still give logs though. 
 https://github.com/glyphx/wtc
I'm going over to try your suggestion now, tyvm.
